server/lib/src/server/delete.rs

```diff
@@ -261,8 +261,8 @@ mod tests {
         assert!(cr.is_ok());
 
         // Delete filter is syntax invalid
-        let de_inv = DeleteEvent::new_internal_invalid(filter!(f_pres("nhtoaunaoehtnu")));
-        assert!(server_txn.delete(&de_inv).is_err());
+        // let de_inv = DeleteEvent::new_internal_invalid(filter!(f_pres("nhtoaunaoehtnu")));
```

Comment:
Given that f_pres doesn't accept a raw string now, should we be testing this with one of the "test" attributes that this object doesn't have?

Reply:
The problem here is that we need to actually reject the invalid attribute name. So provided there is a test attribute name that "will never exist" in schema then yes we can do that.

Reply:
ok I'll work out a way of rejecting it at build time 😄

Reply:
Shouldn't we just have an Attribute that is "CanNeverExist" that yields the gibberish, and then we cfg(test, debug assert) it out? Seems easier ....

Reply:
Weeeellll the Attribute can't exist if it's not in the enum 😄

Reply:
You could also bypass the macro and just create the FC directly.